examples/benchmarks/blas.cpp

```diff
@@ -25,11 +26,18 @@ int main(int argc, char** argv) {
     try {
         int device = argc > 1 ? atoi(argv[1]) : 0;
         setDevice(device);
+
+        std::string dtype(argc > 2 ? argv[2] : "f32");
```

Comment:
It would be nice if you could handle the case were you don't specify the target device.

Reply:
I actually prefer the cli to ask for device id for him to make sure he selects the right gpu... Anyway, feel free to change that later if you prefer something else.